CONTRIBUTING.md

```diff
@@ -141,8 +141,8 @@ on each other), the owner should try to get people aligned by:
   split it up.
 
 If none of the above worked and the PR has been stuck for more than 2 weeks, the
-owner should bring it to the [OpenTelemetry C++ SIG
-meeting](https://zoom.us/j/8203130519). The meeting passcode is _77777_.
+owner should bring it to the OpenTelemetry C++ SIG meeting. See
+[README.md](README.md#Contributing) for the meeting link.
```

Comment:
`README.md#Contributing` or `README.md#contributing` or doesn't matter (uppercase or lowercase)?

Reply:
Seems casing doesn't matter in URL. [this](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/README.md#CoNtrIbutinG) URL works as well in my browser.

Reply:
I don't think so.

https://www.w3.org/TR/WD-html40-970708/htmlweb.html

_URLs in general are case-sensitive (with the exception of machine names). There may be URLs, or parts of URLs, where case doesn't matter, but identifying these may not be easy. **Users should always consider that URLs are case-sensitive.**_

Reply:
Thanks for the sharing. It makes sense for the URL to be case-sensitive as it shows up in many short URLs. Perhaps github does the toleration.